exporter/stackdriverexporter/spandata.go

```diff
@@ -0,0 +1,309 @@
+// Copyright 2020, OpenTelemetry Authors
```

Comment:
Is it part of another PR? https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/238/files

Reply:
Yes as the description says, it depends on that PR to be merged first :)